pkg/scheduler/schedule_one_test.go

```diff
@@ -1979,6 +1979,42 @@ func TestSchedulerSchedulePod(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "test prefilter plugin returning skip",
+			registerPlugins: []st.RegisterPluginFunc{
+				st.RegisterQueueSortPlugin(queuesort.Name, queuesort.New),
+				st.RegisterPreFilterPlugin(
+					"FakePreFilter1",
+					st.NewFakePreFilterPlugin("FakeFilter1", nil, nil),
+				),
+				st.RegisterFilterPlugin(
+					"FakeFilter1",
+					st.NewFakeFilterPlugin(map[string]framework.Code{
+						"node1": framework.Unschedulable,
+					}),
+				),
```

Comment:
do we need these?

Reply:
Here, I want to make sure that all other plugins are executed even if some plugins return skip.

Reply:
I'm fine leaving that tests just to the runtime package, but up to you.

Reply:
I'd like to keep it because if this test have existed in the past PR (reverted one), actually we wouldn't have encountered the bug. 
The runtime package's test can only confirm the behavior of specific extension point (Only either Filter or PreFilter) and it cannot confirm what if the scheduler actually run from PreFilter to Filter.